www/posts/2020-05-25-v2-release-day.md

```diff
@@ -0,0 +1,152 @@
+- [Custom build scripts](/#build-scripts) & [third-party plugins](/#build-plugins) to connect your favorite tools.
+
+For most Snowpack v1.x applications, all you need to do is run the following command and Snowpack 2.0 will walk you through updating any outdated configuration:
+
+```
+npm i snowpack@latest --save-dev
+```
+
+
+<br/>
+
+## Snowpack 2.0: Rethinking Best Practices
+
+!["Rethink established best practices" tweet.](/img/react-snarky-tweet-2.png)
+
+"Rethink established best practices." That was the famous antagonistic tweet directed at the React team back when React & JSX were first announced in 2013.
+
+It had been meant as a snarky burn against the project, but the React team turned it into a rallying cry. Pete Hunt even named [an entire JSConf talk](https://www.youtube.com/watch?v=DgVS-zXgMTk) after it, explaining why "rethinking best practices" was exactly what the web needed at that moment. 7 years later, and Pete was clearly proven right.
+
+**Snowpack is a chance to rethink best practices again, this time around web tooling.** The web has evolved over the last decade, but our build tooling hasn't kept up. The best practices of the previous decade no longer fit the requirements of today.
+
+**Snowpack is rethinking the biggest outdated assumption of all: the idea that web developers NEED bundlers.** Bundling continues to be great production optimization, dating all the way back to the early days of the web. It should continue to be available to those who want it. But... it was only in the last decade (with the rise of Browserify and later, Webpack) that bundling became a *required* part of your development workflow. 
+
+<blockquote class="twitter-tweet" data-conversation="none" data-dnt="true"><p lang="en" dir="ltr">Check out the network tab to see unbundled development in action:<br><br>1️⃣ App.js is changed<br>2️⃣ App.js is refetched via HMR<br>3️⃣ App.js is rebuilt from source<br>✅ Total time: a few milliseconds. <br><br>Zero bundling, rebundling, or multi-file compiling needed for the snappiest update possible</p>&mdash; Pika 📦 (@pikapkg) <a href="https://twitter.com/pikapkg/status/1264245654970232832?ref_src=twsrc%5Etfw">May 23, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+**Bundling during development slows you down.**  It may be hard to see, since most of us haven't done meaningful web development *without* a bundler in ages. But under the hood, bundled dev servers like `webpack-dev-server` & Create React App need to load, build, resolve and bundle your application before you can serve a single file. Smart caching can speed this up somewhat, but it's still common for a dev servers to take 30+ seconds to startup. And that's not even counting the time you spend re-bundling your application after you make a change, which can take another several seconds before you see those changes reflected in the browser.
```

Comment:
I'd remove the "may be hard to see" bits here and restructure a little to make the point concise
```markdown
**Bundling during development slows you down.**
Under the hood, bundled dev servers... Smart caching can speed this up...
```
Maybe also add a short 1-sentence "what we're doing different" to close it out?